### PR TITLE
Layout cleanup

### PR DESCRIPTION
**Pull request by axdanbol**
Issue - # (Put issue number preceeded by hash sign to link to issue)

Description of changes - 
Moved common css value into variables in global style files.
Ensured the html/scss/code is well formatted.

@mentions of the person or team responsible for reviewing proposed changes.

TO-DO checklist before submitting your pull-request - 

- [ ] Are all acceptance criteria met ? 
- [ ] Has the code been documented properly ?
- [ ] Does component css-class structure conform to the Blue Dino style template i.e. top-level element .wrapper, .container and other inside css classes ?
- [ ] Have all checks passed ?
- [ ] Have you tested inter-browser functionality (should work on the latest Chrome, Firefox and Edge browsers) ?

### CODE REVIEW

File: src/app/views/main/main.component.scss [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
+      height: $header-height;
     }
-    .drawer-wrapper {
+
+    .contents-area {
       display: flex;
-      height: calc(100vh - 3.1875rem);
-      margin-left: 4.25rem;
-      .container {
+      height: calc(100vh - #{$header-height});
```
Comment (bherr2) [inline src/app/views/main/main.component.scss]: Should that be a $ instead of a #?

Comment (axdanbol) [reply src/app/views/main/main.component.scss]: It is interpolation, otherwise the css ends up with calc(100vh - $header-height), which is invalid

Comment (axdanbol) [reply src/app/views/main/main.component.scss]: For some reason it can't figure it out on its own

Comment (bherr2) [reply src/app/views/main/main.component.scss]: Huh weird. Ok.

**Pull request merged**